internal/auth0/organization/resource_connection.go

```diff
@@ -96,12 +117,17 @@ func updateOrganizationConnection(ctx context.Context, data *schema.ResourceData
 	api := meta.(*config.Config).GetAPI()
 
 	organizationID := data.Get("organization_id").(string)
-
 	connectionID := data.Get("connection_id").(string)
 	assignMembershipOnLogin := data.Get("assign_membership_on_login").(bool)
 
+	// We need to keep these null if they don't appear in the schema
+	isSignupEnabled := value.Bool(data.GetRawConfig().GetAttr("is_signup_enabled"))
```

Comment:
Same as above comment.

Reply:
same as above response :-)

Reply:
Couldn't we use the `expandOrganizationConnection` func here as well, just like in the create?